src/controllers/SalesEmployeeController.ts

```diff
@@ -0,0 +1,57 @@
+import express from "express";
+import { createSalesEmployee, deleteSalesEmployee, editSalesEmployee, getSalesEmployee, getSalesEmployees } from "../services/SalesEmployeeService";
+
+export const getAllSalesEmployees = async (req: express.Request, res: express.Response): Promise<void> => {
+    return res.render('salesEmployee/list.njk', { salesEmployees: await getSalesEmployees() });
+}
+
+export const getSingleSalesEmployee = async (req: express.Request, res: express.Response): Promise<void> => {
+    const salesEmployee = await getSalesEmployee(req.params.id);
+    if (!salesEmployee) return res.render('errors/404.njk');
+    return res.render('salesEmployee/detail.njk', { salesEmployee });
+}
+
+export const getSingleSalesEmployeeToDelete = async (req: express.Request, res: express.Response): Promise<void> => {
+    const salesEmployee = await getSalesEmployee(req.params.id);
+    if (!salesEmployee) return res.render('errors/404.njk');
+    return res.render('salesEmployee/delete.njk', { salesEmployee });
+}
+
+export const deleteSingleSalesEmployee = async (req: express.Request, res: express.Response): Promise<void> => {
+    const salesEmployee = await getSalesEmployee(req.params.id);
+    try {
+        await deleteSalesEmployee(req.params.id);
+        return res.render('salesEmployee/deleteSuccess.njk');
+    } catch (e) {
+        return res.render('salesEmployee/delete.njk', { salesEmployee, error: e });
+    }
+}
+
+export const getCreateSingleSalesEmployee = async (req: express.Request, res: express.Response): Promise<void> => {
```

Comment:
the naming is very confusing for this, get create doesnt make sense. if we are doing routing it could be render instead of get